en/plugins.rst

```diff
@@ -51,83 +51,52 @@ Manually Autoloading Plugin Classes
 If you install your plugins via ``composer`` or ``bake`` you shouldn't need to
 configure class autoloading for your plugins.
 
-If we were installing a plugin named ``MyPlugin`` manually you would need to
-modify your application's **composer.json** file to contain the following
-information:
+If you create a plugin manually under the ``plugins`` folder then will need to
+tell ``composer`` to refresh its autoloading cache:
```

Comment:
```suggestion
If you create a plugin under the ``plugins`` directory of your application then will need to
tell ``composer`` to refresh its autoloading cache:
```

Reply:
I used the word "manually" to indicate as opposed to baking, since baking a plugin already runs `composer dumpautoload`.

Reply:
there is a word missing after "then"